src/js/core/util/rotaryScrolling.js

```diff
@@ -39,18 +39,41 @@
 				 * Scroll step
 				 * @type {number}
 				 */
-				scrollStep = 40;
+				scrollStep = 40,
+				/**
+				 * Time after bound effect will be hidden.
+				 * type {number}
+				 */
+				bouncingTimeout = 1000;
 
 			/**
 			 * Handler for rotary event
 			 * @param {Event} event Event object
 			 */
 			function rotaryDetentHandler(event) {
+				var scrollview = null,
+					maxScrollY = 0;
+
 				if (element.getAttribute("data-lock-rotary-scroll") !== "true") {
+					scrollview = ns.engine.getBinding(element.parentElement, "Scrollview");
 					if (event.detail.direction === "CW") {
 						element.scrollTop += scrollStep;
+						if (scrollview) {
+							maxScrollY = element.scrollTop + window.innerHeight;
+							scrollview.bouncingEffect.drag(undefined /* ignore horizontal effect */, -maxScrollY);
```

Comment:
This is little strange, this `undefined`as argument. Is this only way?

Reply:
This can not be 0 and not maxScrollX since they will cause to show vertical bouncing which is not we want. I think leaving it as undefined with comment make the code clear.

Reply:
I added new "orientation" parameter to drag(...) method to force only vertical bouncing in case of rotary on Content. This looks quite good but reversing coordinates in Scroller (used by SecrionChanger) prevents us to use bouncing with Content (conditions are not meet).

Alternatively we could call drag() method only when end of scroll is reached (as it's done in Arclistview). This will allow calling drag(0, -maxScrollY). I leave this for future modifications.